napari/layers/utils/layer_utils.py

```diff
@@ -228,6 +228,14 @@ def calc_data_range(data, rgb=False) -> Tuple[float, float]:
     if data.dtype == np.uint8:
         return (0, 255)
 
+    if isinstance(data, np.ndarray) and data.ndim < 3:
+        min_val = _nanmin(data)
+        max_val = _nanmax(data)
+        if min_val == max_val:
+            min_val = 0
+            max_val = 1
```

Comment:
Maybe `min_val, max_val = min_val - 1, max_val + 1`? This allows the range to include the min and max, which I think is important.

Reply:
This repeat behavior from the end of this function. Should I change in both places?